pebbles/views/blueprint_templates.py

```diff
@@ -127,6 +129,22 @@ def put(self, template_id):
         db.session.commit()
 
 
+class BlueprintTemplateCopy(restful.Resource):
+    @auth.login_required
+    @requires_admin
+    def put(self, template_id):
+        template = BlueprintTemplate.query.filter_by(id=template_id).first()
+        if not template:
```

Comment:
http://flask-sqlalchemy.pocoo.org/2.1/api/#flask.ext.sqlalchemy.BaseQuery.get_or_404

A more idiomatic way of doing these things. Nothing too wrong with being very explicit though.